torchtext/prototype/models/t5/t5_transform.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, List
+
+import torch
+import torch.nn as nn
+import torchtext.transforms as T
+from torchtext.data.functional import load_sp_model
+from torchtext.functional import to_tensor
+from torchtext.utils import get_asset_local_path
+
+
+class T5Transform(nn.Module):
+    def __init__(self, sp_model_path: str, max_seq_len: int, eos_idx: int, padding_idx: int):
+        super().__init__()
+
+        self.sp_model = load_sp_model(get_asset_local_path(sp_model_path))
+        self.max_seq_len = max_seq_len
+        self.eos_idx = eos_idx
+        self.padding_idx = padding_idx
+        self.pipeline = T.Sequential(T.Truncate(self.max_seq_len), T.AddToken(token=self.eos_idx, begin=False))
```

Comment:
Curious if you have tried to add self.encode to T.Sequential and see if it works?

Reply:
I have but it does not work. The error is that Sequential only accepts Module subclasses